portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
```

Comment:
This is great, could you please include screenshot of the portfolio to see how it looks

Reply:
![Screenshot 2020-05-28 at 10 25 32 AM](https://user-images.githubusercontent.com/16737842/83173374-ca3da980-a0cd-11ea-9e57-6e0758a11fbf.png)